src/components/containers/ArticleLists/ArticleLists.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Item, Icon } from 'semantic-ui-react';
+
+// components
+import Title from '../../presentationals/Title/Title';
+
+const f16 = {
+  fontSize: '16px',
+};
+
+class ArticleLists extends Component {
+  render() {
+    const { articles } = this.props;
+
+    return (
+      <div className="post-lists">
+        {articles.length === 0 ? (
+          <p>You have no publications</p>
+        ) : (
+          articles.map(article => (
+            <Item.Group key={article.id}>
+              <Item>
+                <Item.Image src="https://react.semantic-ui.com/images/wireframe/image.png" />
+
+                <Item.Content>
+                  <Title content={article.title} className="title-article-lg" />
+                  <Item.Description
+                    style={{
+                      marginBottom: 'auto',
+                    }}
+                  >
+                    <p
+                      style={{
+                        fontSize: '18px',
```

Comment:
Is there a reason why you chose px units here?

Reply:
Converting these to rem values now, thank you for spotting this.